### DISCUSSION TITLE
Add 2013 twitter

### DISCUSSION THREAD

**Discussion by goodoldweb2013** [dimdenGD/OldTwitter#683] [Ideas]
can you guys, add 2013 twitter? it would be cool!

**Comment by haydengriz** [comment]
I doubt that’s going to be done, what did Twitter look like in 2013 again?

**Comment by telnettrauma** [comment]
>  what did Twitter look like in 2013 again?

I found an old article with screenshots of 2013 Twitter: [https://techcrunch.com/2010/09/14/the-new-twitter/](https://techcrunch.com/2010/09/14/the-new-twitter/)

Although it would be cool, I don't think it would be worth it and I don't feel like most modern Twitter features would integrate well with this layout.

**Reply by ikkqqstudio** [reply]
This is a Post About **2010** Twitter! **Not 2013!**
The Twitter logo On The Post Was From **2010! Not 2012!**
